### ISSUE TITLE
Data transforms(center crop/resize) in test/validation set?

### ISSUE THREAD

**Issue by zooyeonii** [boostcampaitech3/level1-image-classification-level1-cv-06#21]
It is common practice to augment data (_Color Jittering, Flip, RandomCrop_, etc) on training data set only. 

But how about _CenterCrop_ or _resize_? Do we also apply these only on  training data? 

I think both can be applied to training and test data set, because those don't have a randomness. 

But especially for _centercrop_, it's confusing to figure out the appropriateness of whether to apply it to test dataset...  If it can be used, what would be the effect?

```
self.transform = transforms.Compose([
            CenterCrop((330, 256)),
            transforms.Resize(resize, Image.BILINEAR),
            transforms.ToTensor(),
            transforms.Normalize(mean=mean, std=std),
        ])
```
